app/site/controllers.py

```diff
@@ -111,9 +111,17 @@ def datapackage_show(publisher, package):
                            datapackageUrl=datapackage_json_url_in_s3,
                            showDataApi=True, jsonDataPackage=dataset,
                            dataViews=dataViews,
+                           publisherName=publisher,
+                           packageName=package,
                            s3_cdn=get_s3_cdn_prefix()), 200
 
 
 @site_blueprint.route("/<publisher>", methods=["GET"])
 def publisher_dashboard(publisher):
-    return render_template("publisher.html")
+    datapackage_list_url = "/api/search/package?q=* publisher:{publisher}".format(publisher=publisher)
+    publisher_profile_url = "/api/profile/publisher/{name}".format(name=publisher)
+    datapackage_list = json.loads(app.test_client().get(datapackage_list_url).data)
+    publisher = json.loads(app.test_client().get(publisher_profile_url).data)
```

Comment:
@Fak3 can you please tell me what is the best way to call the url of other module. How do we pass query parameter and other request attribute e.g. header.